addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -3380,9 +3380,21 @@ export class OdooEditor extends EventTarget {
         if (!rangeContent.hasChildNodes()) {
             return;
         }
+        // find list of ancestors.
```

Comment:
This comment is not very useful because it only talks about the next line which is a `const` named `ancestorList` so we can understand easily what it does. However, the code below with the `LI` and the `break` has no comment to explain why we need a special `break` in that case. It would be a more interesting and helpful comment IMO.